apps/condo/pages/ticket/[id]/pdf.tsx

```diff
@@ -158,7 +159,21 @@ const PdfView = () => {
                     <Col span={6}>
                         <TicketDescriptionField
                             title={ClassifierMessage}
-                            value={get(ticket, ['classifier', 'name'])}
+                            value={
+                                <Breadcrumb separator={<>≫<br/></>}>
+                                    {
+                                        compact([
```

Comment:
useMemo

Reply:
It's irrelevant here, because a pdf version is not going to be changed to be rerendered